src/lib/containers/entity/metadata/EntityModal.tsx

```diff
@@ -62,36 +66,90 @@ export const EntityModal: React.FC<EntityModalProps> = ({
     isVersionable && (entityBundle.entity as VersionableEntity).isLatestVersion!
 
   useEffect(() => {
-    if (show) {
-      rebuildTooltip()
+    rebuildTooltip()
+  })
```

Comment:
Tooltip/modal interaction is weird, so just run rebuild every render. Our rebuildTooltip utility has debounce, so it shouldn't cause any issues.